.github/CODEOWNERS

```
@@ -105,3 +105,19 @@ sycl/test-e2e/KernelFusion @victor-eds @Naghasan @sommerlukas
 
 # Matrix tests
 sycl/test-e2e/Matrix @dkhaldi @YuriPlyakhin @yubingex007-a11y
+
+#Native CPU PI 
+sycl/plugins/native_cpu/ @intel/dpcpp-nativecpu-pi-reviewers
+sycl/test/check_device_code/native_cpu/ @intel/dpcpp-nativecpu-pi-reviewers 
+sycl/test/native_cpu/ @intel/dpcpp-nativecpu-pi-reviewers 
+clang/include/clang/Basic/SYCLNativeCPUHelpers.h @intel/dpcpp-nativecpu-pi-reviewers 
+clang/test/CodeGenSYCL/native_cpu_basic.cpp @intel/dpcpp-nativecpu-pi-reviewers 
+clang/test/Driver/sycl-native-cpu-fsycl.cpp @intel/dpcpp-nativecpu-pi-reviewers 
+clang/test/Driver/sycl-native-cpu-warn.cpp @intel/dpcpp-nativecpu-pi-reviewers 
+clang/test/Driver/sycl-native-cpu.cpp @intel/dpcpp-nativecpu-pi-reviewers 
```

I'm a little bit concerned about listing each test name explicitly here. I would rather come up with some naming scheme we can match to allow the team to add new tests w/o modifying CODEOWNERS file each time (it can easily be missed).
Let's say if we agree that all the tests will start with `sycl-native-cpu` or `native_cpu` we can use something like:
```suggestion
clang/test/CodeGenSYCL/native_cpu*.cpp @intel/dpcpp-nativecpu-pi-reviewers 
clang/test/Driver/sycl-native-cpu*.cpp @intel/dpcpp-nativecpu-pi-reviewers 
```

@intel/dpcpp-nativecpu-pi-reviewers, what do you think?

---

Yeah good point, we'll make sure to follow the naming convention